tools/setup/emoji/emoji_names.py

```diff
@@ -455,7 +455,7 @@
     '1f409': {'canonical_name': 'dragon', 'aliases': []},
     '1f432': {'canonical_name': 'dragon_face', 'aliases': []},
     '1f335': {'canonical_name': 'cactus', 'aliases': []},
-    '1f384': {'canonical_name': 'holiday_tree', 'aliases': []},
+    '1f384': {'canonical_name': 'holiday_tree', 'aliases': ['vacation']},
```

Comment:
I tihnk this should actually go on the palm tree, as you suggested in your chart.

Reply:
```suggestion
   '1f334': {'canonical_name': 'palm_tree', 'aliases': ['vacation']},
```
done